ydb/docs/en/core/integrations/liquibase.md

```diff
@@ -0,0 +1,551 @@
+| `Bool`                                  | `true` or `false`                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| `Int8`, `Int16`, `Int32`, `Int64`       | A signed integer                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| `Uint8`, `Uint16`, `Uint32`, `Uint64`   | An unsigned integer                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `Text`, `Bytes`, `Json`, `JsonDocument` | Represent as text                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| `Float`, `Double`, `Decimal(22, 9)`     | A real number                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| `Interval`                              | [Format](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/Duration.html#parse(java.lang.CharSequence)) `java.time.Duration`, prefix `PT` is optional                                                                                                                                                                                                                                                                                       |
```

Comment:
This `PT` thing is part of ISO8601 too, and there might be days/weeks/years between P and T, and intervals can be negative, so it's not just a prefix. So if it follows the standard it's better to refer to it instead of Java docs. Please clarify what exactly is optional. Are there any additional limitations like in YDB itself?